themes/af-magic.zsh-theme

```diff
@@ -27,7 +27,7 @@ else
 fi
 
 # git settings
-ZSH_THEME_GIT_PROMPT_PREFIX="$FG[075](branch:"
+ZSH_THEME_GIT_PROMPT_PREFIX="($FG[078]branch:"
 ZSH_THEME_GIT_PROMPT_CLEAN=""
 ZSH_THEME_GIT_PROMPT_DIRTY="$my_orange*%{$reset_color%}"
-ZSH_THEME_GIT_PROMPT_SUFFIX="$FG[075])%{$reset_color%}"
+ZSH_THEME_GIT_PROMPT_SUFFIX="$FG[078])%{$reset_color%}"
```

Comment:
The parenthesis before 'branch' is a different colour than the `SUFFIX` parenthesis.

The following example makes sure they're both the same colour:
```ZSH_THEME_GIT_PROMPT_SUFFIX=")$FG[078]%{$reset_color%}"```